x-pack/plugins/watcher/__jest__/client_integration/watch_create_json.test.ts

```diff
@@ -3,6 +3,13 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
 import '../../../es_ui_shared/console_lang/mocks';
```

Comment:
```suggestion
 */

import '../../../es_ui_shared/console_lang/mocks';
```

How did this get here? Are the eslint rules broken?

Reply:
@spalger I don't think Eslint is broken. I must have copied and pasted something messed up the comments. I will fix that.